packages/apollo-server/README.md

```diff
@@ -0,0 +1,88 @@
+# `@appsignal/apollo-server`
+
+![npm (scoped)](https://img.shields.io/npm/v/@appsignal/apollo-server) ![npm peer dependency version (scoped)](https://img.shields.io/npm/dependency-version/@appsignal/apollo-server/peer/apollo-server)
+
+The AppSignal for Node.js integration for Apollo GraphQL (`apollo-server`) v2.0.0+.
+
+## Installation
+
+First, [sign up for an AppSignal account][appsignal-sign-up] and add both the `@appsignal/nodejs` and `@appsignal/apollo-server` packages to your `package.json`. Then, run `yarn install`/`npm install`.
+
+You can also add these packages to your `package.json` on the command line:
+
+```bash
+yarn add @appsignal/nodejs @appsignal/apollo-server
+npm install --save @appsignal/nodejs @appsignal/apollo-server
+```
+
+You can then import and use the package in your app. 
+
+## Usage
+
+The module includes an Apollo Server plugin for automatically instrumenting the resolvers of your application.
+
+```js
+// ENSURE APPSIGNAL IS THE FIRST THING TO BE REQUIRED/IMPORTED
+// INTO YOUR APP!
```

Comment:
I wouldn't document things with all caps. It's not really the tone we want to set.